src/core/Util.js

```diff
@@ -239,3 +239,26 @@ export function cancelAnimFrame(id) {
 		cancelFn.call(window, id);
 	}
 }
+
+function isNumeric(str) { // https://stackoverflow.com/a/175787/2520247
+	return !isNaN(str) && !isNaN(parseFloat(str));
+}
+
+// keep it private intentionally, because such function implementation
+// heavily depends on it's particular application and unable to suit all the cases
+export function _checkNumber(a) {

```

Comment:
I'm going to nitpick on the semantics here. This doesn't *check* for a `Number`, but rather *casts* the parameter into a `Number`.
I'll argue that this function should instead be called `castFinite`. No real reason to keep this private or undocumented IMO.

Reply:
We had already the same discussion [here](https://github.com/Leaflet/Leaflet/pull/7768#discussion_r747495070) with no result.

Yes the function is casting to a number but also throws an error if it is not a number `throw new Error('Number expected');`. I personally would like to keep `Number` instead of `Finite` but if we call it `cast` or `check` doesn't matter for me.